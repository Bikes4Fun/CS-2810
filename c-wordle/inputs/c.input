st(a)[e](m)
(s)t[e](a)m
steam
[s](t)[e](a)[m]
