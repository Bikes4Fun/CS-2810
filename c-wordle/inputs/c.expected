Here are the 4 guesses that I read:
st(a)[e](m)
(s)t[e](a)m
steam
[s](t)[e](a)[m]
